README.md

```diff
@@ -254,6 +254,22 @@ or
 pytest test/integration/trino.dbtspec
 ```
 
+### Release process
+
+Before doing a release dbt's version requires updating.
+In order to bump dbt-trino and dbt-core version run:
+
+```sh
+bumpversion --config-file .bumpversion-dbt.cfg patch --new-version <new-version>
+bumpversion --config-file .bumpversion.cfg patch --new-version <new-version> --allow-dirty
+```
+
+Additionally bump dbt-core version in dbt testing image `docker/dbt/Dockerfile`.
```

Comment:
Can we do all these steps  bumping steps with only *1* command?

Reply:
With bumpversion I think it cannot be done. We can have a script which does all the above. But I don't think that is a big deal to update it in one more place.

Reply:
👍